style.css

```diff
@@ -0,0 +1,89 @@
+/* style sheet for rocket science home */
+
+body {
+        background-image:url("background.png");
+        background-repeat:repeat-y;
```

Comment:
I don't think this is correct. If we were to add a hundred lines of extra content to this, we would prefer the background stay black below the bottom of the image. I think `no-repeat` and `background-color: #000;` (which will show up anywhere that isn't covered by the image) is what we're looking for!